ui/qt-lib/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs

```diff
@@ -180,10 +194,15 @@ handleWalletInfoEvent UiLangFace{..} ev = do
               UiSelectionWallet UiWalletInfo{..} -> (uwiLabel, uwiBalance, True)
               UiSelectionAccount UiAccountInfo{..} -> (uaciLabel, uaciBalance, False)
 
-      whenJust itemName $ QLabel.setText itemNameLabel . toString . toUpper
+      QLabel.setText itemNameLabel . toString . toUpper . fromMaybe "" $ itemName
```

Comment:
I believe this `Maybe` is pointless and misleading. How can wallet / account not have a name? I'd rather drop it.